db/schema.rb

```diff
@@ -88,6 +88,9 @@
     t.index ["parent_activity_id"], name: "index_budgets_on_parent_activity_id"
   end
 
+  create_table "data_migrations", primary_key: "version", id: :string, force: :cascade do |t|
```

Comment:
I'm unsure how this has got in as a new addition! [On develop right now this table is not in the schema](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/db/schema.rb) which is unexpected.

We've been using the data_migrate gem since April https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/commit/12f921d6e9dd5311817de06c547c05f944c0680f and there hasn't been a migration needed.

Any idea where this came from, what it does and if we need it?

Reply:
No and yikes, I will recreate my db to make sure this doesn't happen again